appdatabase/migrations/sql/1678275064_add_created_at_to_saved_addresses.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE saved_addresses ADD COLUMN created_at INT DEFAULT 0;
+UPDATE saved_addresses SET created_at = 0;
```

Comment:
This should not be needed if we have a default value, right?

Reply:
I believe by default the new value is not set to old record